app/scraping.py

```diff
@@ -1,19 +1,39 @@
 from bs4 import BeautifulSoup
 import os
+import requests
+from PIL import Image
+from io import BytesIO
 
 
 def extract_images(soup: BeautifulSoup) -> list:
     images = []
     n_images = int(os.getenv("N_IMAGES", 4))  # Default to 4 if not set
+    min_width = int(os.getenv("MIN_IMAGE_WIDTH", 0))
```

Comment:
Hardcode this is as a global variable for now

Reply:
Sure, I can update the code to hardcode the minimum width and height as global variables. I'll make the changes and push the update shortly. Thank you for the feedback!

Reply:
Sure, I can update the code to use a global variable for the minimum image dimensions. I'll make the necessary changes. Thank you for the feedback!